### PR TITLE
Improved VL study forms and reports and fixed some bugs

### PR DESCRIPTION
**Pull request by Pkom17**

### CODE REVIEW

File: src/main/java/org/openelisglobal/image/service/ImageServiceImpl.java
Comment (CalebSLane) [inline src/main/java/org/openelisglobal/image/service/ImageServiceImpl.java:1]: If we are going to catch the generic exception, we should do a LogEvent.LogErrorStack to make sure we can trace the error if it happens

File: src/main/resources/static/scripts/entryByProjectUtils.js
Comment (CalebSLane) [inline src/main/resources/static/scripts/entryByProjectUtils.js:1]: what is the value 1251 supposed to be? if it's an id is it going to be static across all instances? if not, can it be declared as a named variable to give it more context?

File: src/main/resources/static/scripts/neon/entryByProjectUtils.js
Comment (CalebSLane) [inline src/main/resources/static/scripts/neon/entryByProjectUtils.js:1]: what is the value 1251 supposed to be? if it's an id is it going to be static across all instances? if not, can it be declared as a named variable to give it more context?

File: src/main/java/org/openelisglobal/patient/controller/BasePatientEntryByProject.java
Comment (CalebSLane) [inline src/main/java/org/openelisglobal/patient/controller/BasePatientEntryByProject.java:1]: what is distinct about genders_2 from genders?

**Review by CalebSLane — CHANGES_REQUESTED** [review]

File: src/main/java/org/openelisglobal/image/service/ImageServiceImpl.java
Comment (Pkom17) [reply src/main/java/org/openelisglobal/image/service/ImageServiceImpl.java:1]: Okay... LogEvent has been added

File: src/main/resources/static/scripts/entryByProjectUtils.js
Comment (Pkom17) [reply src/main/resources/static/scripts/entryByProjectUtils.js:1]: 1251 is the dictionary ID for "No" response  in the dictionary table and it is supposed to be static accross instances. 
it is used here to check "No" response by default in the study forms

File: src/main/resources/static/scripts/neon/entryByProjectUtils.js
Comment (Pkom17) [reply src/main/resources/static/scripts/neon/entryByProjectUtils.js:1]: 1251 is the dictionary ID for "No" response  in the dictionary table and it is supposed to be static accross instances. 
it is used here to check "No" response by default in the study forms

File: src/main/java/org/openelisglobal/patient/controller/BasePatientEntryByProject.java
Comment (Pkom17) [reply src/main/java/org/openelisglobal/patient/controller/BasePatientEntryByProject.java:1]: finally commented out the first one as it is not used elsewhere.

**Review by CalebSLane — APPROVED** [review]

**Pull request merged**